FirebaseVertexAI/Tests/Unit/ChatTests.swift

```diff
@@ -39,9 +39,9 @@ final class ChatTests: XCTestCase {
 
     // Skip tests using MockURLProtocol on watchOS; unsupported in watchOS 2 and later, see
     // https://developer.apple.com/documentation/foundation/urlprotocol for details.
-    guard #unavailable(watchOS 2) else {
```

Comment:
Note: This was causing unit tests to get skipped on non-watchOS platforms as well. Since  we only support relatively modern versions, I updated this to skip on watchOS regardless of version with `#if os(watchOS)`.